### DISCUSSION TITLE
Would a USB header splitter + Razer PWM Fan Controller work?

### DISCUSSION THREAD

**Discussion by michaellashinsky** [EvanMulawski/FanControl.Razer#4] [Q&A] [answered]
Title pretty much says it all.  I am not sure how the USB chaining would affect this or the Fan Control application.

I have 1 Razer PWM Fan controller and 2 Razer ARGB controllers. While I know 2 ARGB controllers can't work for Razer Synapse, I can make them work with SignalRGB.  This being said, I want to keep the ARGB controllers on separate USB ports and tie the PWM controller into a splitter with one ARGB controller.

I intend on using this [splitter](https://www.amazon.com/gp/product/B08N4QVMJ5l) to plug 1 ARGB controller and the PWM fan controller, into a USB2.0 header on the mobo. The other ARGB controller would occupy the other USB header. These would be on an ASUS x570-E Gaming motherboard.

I can post back if it works, I am just not sure if splitters would work like a hub and if that hub would work with both this plugin and the parent application.

**Comment by arlo-infinity** [comment]
@michaellashinsky I can't remember if the PWM Fan controller and/or the ARGB controllers uses a full USB 2.0 header or only half of it.   If BOTH the PWM Fan controller and the ARGB controller use 4 wires only, then you can hook them up with a simple splitter and they should both function as the splitter splits the 8 pins of the USB 2.0 header between them (one row of 4 each).  However, if either of the PWM Fan controller, the ARGB controller, or whatever device is connected uses all 8 pins then you need to instead have an internal USB hub rather than a splitter.  I can confirm that the PWM Fan controller works for me using a hub if you are wondering.

**Reply by michaellashinsky** [reply]
Thanks for the response, I think it requires all 8 pins. I tried with a simple splitter, and ended up sending it back as none of the devices were recognized after it was installed. 

I can confirm that [this](https://www.amazon.com/dp/B09YL9JSK7) will 100% not work.

**Reply by michaellashinsky** [reply, marked as answer]
I just got my usb hub installed.  I loaded the plugin (after unblocking it) and it detected everything.

To answer my own question, yes a usb hub feeding the Razer PWM signal to the motherboard will work.
